docs/Makefile

```diff
@@ -37,7 +37,6 @@ help:
 
 clean:
 	-rm -rf $(BUILDDIR)
-	-rm -rf api
```

Comment:
Why is this line removed? This is a generated folder, so a clean up should delete this.